library/tasks/HISAT2.wdl

```diff
@@ -240,8 +241,11 @@ task HISAT2SingleEnd {
       --new-summary --summary-file "${output_basename}.log" \
       --met-file ${output_basename}.hisat2.met.txt --met 5 \
       --seed 12345 \
+      -k 10 \
```

Comment:
Is it possible to record the logic behind setting these parameters? I know the existing WDL sets a bad precedent, but it's nice to be able to understand why, e.g. `k=10`.

Reply:
So there is a comment in the WDL above, and it turns out `-k 10` is just the default behavior or HISAT2. I can at least explain that this is copied from the PE task.